subprojects/language-native/src/test/groovy/org/gradle/language/cpp/plugins/CppExecutablePluginTest.groovy

```diff
@@ -66,7 +66,7 @@ class CppExecutablePluginTest extends Specification {
         then:
         def compileDebugCpp = project.tasks.compileDebugCpp
         compileDebugCpp instanceof CppCompile
-        compileDebugCpp.includes.files == [project.file("src/main/headers")] as Set
+        compileDebugCpp.includes.files.first() == project.file("src/main/headers")
```

Comment:
❓ Which tool chain does this end up picking? just the default for the platform?

Reply:
What do you mean? The system includes for the toolchain set on the compile task for `DefaultNativePlatform("current")` will be added as includes.